### DISCUSSION TITLE
Using the project

### DISCUSSION THREAD

**Discussion by pipalgorithm** [prankshaw/Election-Analytica#4] [Q&A] [answered]
How to use and run this project for further development and research

**Comment by prankshaw** [comment, marked as answer]
The project in itself is build using Jupyter notebook. To study and further use this project
1. Download dependencies to run Jupyter notebook locally (preferred Anaconda)
2. Clone the project and start running the main Notebook.
3. Interact with each cell of code sequentially
4. Develop further and add to this project.

Thank you
